app/helpers/administrate/application_helper.rb

```diff
@@ -16,5 +16,21 @@ def display_resource_name(resource_name)
           default: resource_name.to_s.pluralize.titleize,
         )
     end
+
+    def svg_tag(asset, options = {})
+      svg_id = options.fetch(:svg_id)
+
+      svg_attributes = {
+        "xlink:href" => "#{asset_url(asset)}##{svg_id}"
+      }
+      svg_attributes[:width] = options[:width] if options[:width]
+      svg_attributes[:height] = options[:height] if options[:height]
+
+      content_tag :svg,
+        xmlns: 'http://www.w3.org/2000/svg'.freeze,
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.